pulp_2to3_migration/app/plugin/deb/pulp2_models.py

```diff
@@ -0,0 +1,174 @@
+    size = mongoengine.IntField()
+    relativepath = mongoengine.StringField()
+
+    # Relational fields:
+    # Note: These are intended for structured relationship information. Raw
+    # relationship field strings as used in Debian control files and Packages
+    # indicies are stored in the control_fields dict instead.
+    breaks = mongoengine.DynamicField()
+    conflicts = mongoengine.DynamicField()
+    depends = mongoengine.DynamicField()
+    enhances = mongoengine.DynamicField()
+    pre_depends = mongoengine.DynamicField()
+    provides = mongoengine.DynamicField()
+    recommends = mongoengine.DynamicField()
+    replaces = mongoengine.DynamicField()
+    suggests = mongoengine.DynamicField()
+
+    # List of relational fields:
+    REL_FIELDS = ['breaks', 'conflicts', 'depends', 'enhances', 'pre_depends',
+                  'provides', 'recommends', 'replaces', 'suggests']
+
+    # The control file fields dict:
+    # Note: This stores a dict of strings as used within the python-debian
+    # library. This allows us to retain all control file information, even for
+    # fields not explicitly supported by pulp_deb.
+    control_fields = mongoengine.DynamicField()
+
+    # Remaining control file fields:
+    # Note: With the addition of the control_fields dict, these fields contain
+    # redundant information.
+    source = mongoengine.StringField()
+    maintainer = mongoengine.StringField()
+    installed_size = mongoengine.StringField()
+    section = mongoengine.StringField()
+    priority = mongoengine.StringField()
+    multi_arch = mongoengine.StringField()
+    homepage = mongoengine.StringField()
+    description = mongoengine.StringField()
+    original_maintainer = mongoengine.StringField()
+
+    # Fields retained for backwards compatibility:
+    _ns = mongoengine.StringField(required=True, default=meta['collection'])
+    _content_type_id = mongoengine.StringField(required=True, default=TYPE_ID)
+
+    # A dict translating all control file field names from this class into their
+    # python-debian (deb822) equivalent.
+    # Note: Fields not found in control files are handled separately.
+    TO_DEB822_MAP = dict(
+        name="Package",
+        version="Version",
+        architecture="Architecture",
+        breaks="Breaks",
+        conflicts="Conflicts",
+        depends="Depends",
+        enhances="Enhances",
+        pre_depends="Pre-Depends",
+        provides="Provides",
+        recommends="Recommends",
+        replaces="Replaces",
+        suggests="Suggests",
+        source="Source",
+        maintainer="Maintainer",
+        installed_size="Installed-Size",
+        section="Section",
+        priority="Priority",
+        multi_arch="Multi-Arch",
+        homepage="Homepage",
+        description="Description",
+        original_maintainer="Original-Maintainer",
+    )
+
+
+class DebComponent(ContentUnit):
+    """
+    This unittype represents a deb release/distribution component
```

Comment:
Nit-pick. For the sake of consistency, it would be good to add period at the end of the doc strings. Here and in other places as well.

Reply:
Done.